### DISCUSSION TITLE
如何让进度条显示剩余时间?

### DISCUSSION THREAD

**Discussion by sanaex** [hooke007/mpv_PlayKit#333] [Q&A 问答] [answered]
如何让进度条显示剩余时间, 而不是总时间?

使用的版本是
https://github.com/hooke007/MPV_lazy/releases/download/20230630/mpv-lazy-20230630.exe

**Comment by hooke007** [comment, marked as answer]
https://github.com/hooke007/MPV_lazy/blob/c2db14f04db4bfd467e97e4a2351363595a5c6e3/portable_config/script-opts.conf#L543
